packages/sdk-react/src/core/hooks/useFormPersist.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { SetFieldValue } from 'react-hook-form';
+
+// Persists form state between page reloads
+
+export const useFormPersist = (
+  name: string,
+  getValues: () => { [key: string]: any },
+  setValue: SetFieldValue<any>
+) => {
+  useEffect(() => {
+    const storage = window.sessionStorage;
+    const str = storage.getItem(name);
+
+    if (str) {
+      const values = JSON.parse(str);
```

Comment:
For a JSON.parse it's agood practise to wrap it with try catch

Reply:
Thank you for the suggestion. I updated the code.